packages/network/src/logic/node/NetworkNode.ts

```diff
@@ -16,37 +16,28 @@ export class NetworkNode extends Node {
         this.extraMetadata = metadata
     }
 
-    publish(streamMessage: StreamMessage): void {
+    publish(streamMessage: StreamMessage): void | never {
+        const streamPartId = streamMessage.getStreamPartID()
+        if (this.isProxiedSubscription(streamPartId)) {
```

Comment:
Should we have similar check in `subscribe()` (and `subscribeAndWaitForJoinOperation()`)?

Reply:
I'll this as it's good for clarity